deployment.yml

```diff
@@ -65,10 +65,14 @@ spec:
     metadata:
       labels:
         app: model-service
+      annotations:
+        prometheus.io/scrape: "true"
```

Comment:
Regarding `Prometheus`

If I understand correctly, you:
1. Removed the `ServiceMonitor` custom resource
2. Added `prometheus.io/scrape` annotations in the `Deployement`

And the `prometheus.io/scrape` annotations essentially replace the `ServiceMonitor` custom resource. So now Prometheus can discover our deployed Pods through the annotations that they have. Correct?